### ISSUE TITLE
pushlist 'Listname' 'found' doesn't appear to push serial in the list

### ISSUE THREAD

**Issue by ngyoung** [jsebold666/RazorEnhanced#26]
Pulled this from a larger working script.  The relevant portion of this searches for runebooks in my backpack and adds any with "AV" in their title to a list.  When trying to clickobject, useobject, or even set an alias from the list it returns a Script Error:Invalid Serial.
```
//UOS
//unsetalias 'home'
//unsetalias 'overstockbag'
removelist 'AVbooks'
if not listexists 'AVbooks'
  createlist 'AVbooks'
  clearjournal
  while findtype 0x22c5 'any' 'backpack'
    clickobject 'found'
    waitforjournal 'runebook' 1500
    pause 250
    if injournal 'AV' 'system'
      pushlist 'AVbooks' 'found'
      headmsg 'book found'
      clearjournal
    endif
    pause 300
    ignoreobject 'found'
  endwhile
  clearignorelist
else
  sysmsg 'runebook list found. Starting from here'
endif
clickobject 'AVbooks[0]'
useobject 'AVbooks[0]'

```

**Comment by ngyoung** [comment]
Also tried to do this
`setalias 'test' 'AVbooks[0]'`
and looked into Inspect Alias.  test is listed but with 0x00000

**Comment by credzba** [comment]
I tried to reproduce the AV issue, but runebooks always just put "Runebook" into my journal.

**Comment by credzba** [comment]
resolved in 8.0.28
